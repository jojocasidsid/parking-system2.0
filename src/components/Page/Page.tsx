import React from 'react'
import { Helmet } from 'react-helmet'

interface props {
	title: string
	children: React.ReactNode
}

const Page = ({ title, children }: props) => (
	<>
		<Helmet>
			<title>Ayala: {title}</title>
		</Helmet>
		{children}
	</>
)

export default Page
