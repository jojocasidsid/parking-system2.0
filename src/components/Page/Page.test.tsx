import React from 'react'
import renderer from 'react-test-renderer'

import Page from './Page'

describe('Page Component', () => {
	it('should render page component without crashing', () => {
		const tree = renderer.create(
			<Page title='Test title'>
				<h1>Test Header</h1>
			</Page>
		)

		expect(tree).toMatchSnapshot()
	})
})
