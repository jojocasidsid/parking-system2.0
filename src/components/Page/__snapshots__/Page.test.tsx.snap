// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Component should render page component without crashing 1`] = `
<h1>
  Test Header
</h1>
`;
